src/components/Chloropleth.jsx

```diff
@@ -167,15 +168,16 @@ const Chloropleth = (props) => {
     }
 
     const features = tiles.features.map(f => {
-      const { lad19cd } = f.properties
```

Comment:
Like this change a lot, it was always way too specific. Will allow is to use the 2020 boundaries I guess?

Reply:
Yeah for the moment one has to tinker with the actual JSON, but in due course we could define the accessor instead

Reply:
ah right, interesting!